virttest/utils_net.py

```diff
@@ -1704,21 +1723,7 @@ def restart_guest_network(
     :param timeout: timeout value for command.
     """
     if os_type == "linux":
-        dhcp_clients = [("dhclient", "-r"), ("dhcpcd", "-k")]
```

Comment:
Are these commands equivalent? If so, please mention it in the commit message and bring some proof that you tested that.